### DISCUSSION TITLE
nuxt 3 supabase form value 500 error

### DISCUSSION THREAD

**Discussion by edwardz8** [supabase/supabase#7557] [Questions] [answered]
i have a project with nuxt 3 and supabase where you can submit an email via a form input and locally it works fine and the value is read in the supabase db but when deployed to netlify i run into an issue ( it's a 500 error ).

i built this exact thing a few months back and it worked great but maybe it's something with nuxt 3 or perhaps it's supabase or maybe it's netlify.

i created a stackblitz to demonstrate the project if you are able to take a look but you would have to create your own .env variables and deploy the project to see the issue.

thanks for any direction anyone is able to provide!

https://stackblitz.com/github/edwardz8/nuxt3-rosterbox-news?file=README.md

https://github.com/edwardz8/nuxt3-rosterbox-news

**Comment by silentworks** [comment, marked as answer]
Can you provide some information on what the **Subscriber** table schema should look like, please?

**Reply by edwardz8** [reply]
id ( int8 ) and email (varchar)

**Reply by silentworks** [reply]
I've created the table and deployed this project to Netlify and I'm not getting any errors at all.

**Reply by edwardz8** [reply]
you can see the emails in the table?

this is my project url: https://rosterbox.xyz/ and you can see the error in the console.

**Reply by edwardz8** [reply]
i'm curious what i'm missing in either netlify or supabase because if the only thing you did was clone the project i provided then i must be missing something in either netlify or supabase. but it's weird because it works locally but when deployed the form is not submitted.

**Reply by silentworks** [reply]
Yeah I just cloned it and deployed it to Netlify. I can see the emails in the table. Did you set your env variables in Netlify?

**Reply by silentworks** [reply]
I can't see what the actual error is as you are passing data to a server side function which wouldn't pass the error up. Just check the logs on netlify and it should tell you exactly what is wrong.

**Reply by edwardz8** [reply]
i had not passed the env variables. that fixed it. thanks for the direction.
